v23.1/transaction-retry-error-reference.md

```diff
@@ -63,19 +103,23 @@ This "conversational" design means that there is no way for the server to always
 - [ABORT_REASON_TIMESTAMP_CACHE_REJECTED](#abort_reason_timestamp_cache_rejected)
 - [injected by `inject_retry_errors_enabled` session variable](#injected-by-inject_retry_errors_enabled-session-variable)
 
+Each transaction retry error listed includes an example error as it would appear from the context of the client, a description of the circumstances that cause that error, and specific guidance for addressing the error.
+
 ### RETRY_WRITE_TOO_OLD
 
 ```
 TransactionRetryWithProtoRefreshError: ... RETRY_WRITE_TOO_OLD ...
 ```
 
+_Error type_: Serialization error
+
 _Description_:
 
 The `RETRY_WRITE_TOO_OLD` error occurs when a transaction _A_ tries to write to a row _R_, but another transaction _B_ that was supposed to be serialized after _A_ (i.e., had been assigned a higher timestamp), has already written to that row _R_, and has already committed. This is a common error when you have too much contention in your workload.
 
 _Action_:
 
-1. Retry transaction _A_ as described in [client-side retry handling](transactions.html#client-side-intervention).
+1. Retry transaction _A_ as described in [client-side retry handling](#client-side-retry-handling).
 1. Design your schema and queries to reduce contention. For more information about how contention occurs and how to avoid it, see [Transaction Contention](performance-best-practices-overview.html#transaction-contention). In particular, if you are able to [send all of the statements in your transaction in a single batch](transactions.html#batched-statements), CockroachDB can usually automatically retry the entire transaction for you.
```

Comment:
Can't we just link to this section? https://deploy-preview-16280--cockroachdb-docs.netlify.app/docs/dev/transaction-retry-error-reference.html#minimize-transaction-retry-errors

Users might think "single batch" is the only mitigation.

Reply:
Also we should call out/emphasize the SELECT FOR UPDATE mitigation